mw-webapp/src/component/inProgress/InProgress.module.scss

```diff
@@ -0,0 +1,16 @@
+.container {
+  width: 0%;
+  position: absolute;
+  top: 0;
+  left: 0;
+  height: 4px;
+  background-color: white;
```

Comment:
You can dont add styles or if you want to use it then we use global vars for colors, fonts paddings, etc
So or delete styles or move somw prperties to global